lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -140,6 +142,26 @@ def scrub(transcript)
 
       private
 
+      def add_external_three_ds(params, payment_method, options)
+        return unless three_ds = options[:three_d_secure]
+
+        params[:'3DS'] = {
+          program_protocol: three_ds[:version][0],
+          directory_server_transaction_id: three_ds[:ds_transaction_id],
+          cardholder_name: payment_method.name,
+          card_number: payment_method.number,
+          exp_date: format_exp_date(payment_method.month, payment_method.year),
+          cvv: payment_method.verification_value,
+          xid: three_ds[:acs_transaction_id],
+          cavv: three_ds[:cavv],
+          wallet_provider_id: 'NO_WALLET',
```

Comment:
Do we know this is always true for 3ds?

Reply:
According to docs here https://developer.payeezy.com/payeezy-api/apis/post/transactions-7, is true, please let me know if you think otherwise.

Reply:
@jherreraa  It does reference MASTER_PASS and VISA_CHECKOUT... but I'm not sure if those are necessarily entailed just by card brand. I don't recall anything like this for most other 3DS integrations.  In any case, it seems optional, so I'm comfortable with the default as-is, but we may need to address this later.